builtin/providers/google/resource_container_cluster.go

```diff
@@ -25,8 +25,9 @@ func resourceContainerCluster() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"master_auth": &schema.Schema{
 				Type:     schema.TypeList,
-				Required: true,
+				Optional: true,
 				ForceNew: true,
+				MaxItems: 1,
```

Comment:
Are you running the acceptance tests or are you just looking at the travis build output? The build passes but the acceptance tests didn't when I ran them (https://github.com/hashicorp/terraform/blob/master/.github/CONTRIBUTING.md#running-an-acceptance-test)